_posts/2021-03-12-jdk16-g1-parallel-gc-changes.md

```diff
@@ -6,53 +6,57 @@ date:   2021-03-12 16:39:24 +0200
 tags: [GC, G1, Parallel, JDK 16, Performance]
 ---
 
-Now that JDK 16 release date is very close (2021/03/16 - next Tuesday), there is some time to recap significant changes in Hotspot's stop-the-world garbage collectors - G1 and Parallel GC.
+Now that JDK 16 is released, it is time for a recap of the most significant changes in Hotspot’s stop-the-world garbage collectors - G1 and Parallel GC.
```

Comment:
Well, technically JDK 16 releases tomorrow, and the post is from 2021/03/12, but I'll take the other suggestions.

Not sure how to best manage these really nice suggestions wrt to github - I will take them over manually and reference this pull request.